.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: python
 python:
- - "2.7"
+ - 2.7
+env:
```

Comment:
Would it be alright to add blank lines before each life cycle stage? I think it would make the file contents a little more readable, and conform more closely with the style of the other configuration files.